CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 ## Next Release
 
+### Remove Support for targeting Python 3.8
+
+Mypy now requires `--python-version 3.9` or greater. Support for only Python `3.8` is
+fully removed now. Given an unsupported version, mypy will now default to the oldest
```

Comment:
Can you mention that typeshed stopped supporting 3.8, so mypy can't support it either. And also mention is that 3.8 reached End of Life in Oct 2024?